index.html

```diff
@@ -54,13 +54,19 @@ <h2 class="major">Intro</h2>
 								<h2 class="major">Projects</h2>
 								<span class="image main"><img src="images/pic02.gif" alt="" /></span>
 									
-								<p></p>
+								<p>Here are some Lambda School Projects I have worked on.</p>
+								<ul>
+									<li><a href="https://github.com/Carl-Stanley/responsive-web-design-I">A project involving Responsive Web Design.</a></li>
+									<li><a href="https://github.com/Carl-Stanley/UI-III-Flexbox">A project demonstrating FLEXBOX.</a></li>
+									<li><a href="https://github.com/Carl-Stanley/User-Interface">A basic user interface Web project.</a></li>
```

Comment:
You can deploy your existing projects with github pages so you can link them with the actual/live website running 😉